src/oid.rs

```diff
@@ -18,6 +18,7 @@
 //! thing done with oids in your code. The `oid!` macro can be used in expression positions for
 //! this purpose. For example
 //! ```
+//! extern crate alloc;
```

Comment:
Does this means that every crate using the `oid!` macro has to import `alloc`? That would be annoying.
Or does this concern only `no_std` crates?

Reply:
If this is only for `no_std` crates, we may want to hide this line in the example (use `//! #` instead of `//!`)

Reply:
> Does this means that every crate using the `oid!` macro has to import `alloc`? That would be annoying.
> Or does this concern only `no_std` crates?

Yes, you have grasped the point. The original implementation requires users to manually introduce alloc crates. But after playing with the `proc_macro` all night, I finally found a solution.

Reply:
> If this is only for `no_std` crates, we may want to hide this line in the example (use `//! #` instead of `//!`)

Now there is no need to reflect the alloc crate in the docs, finally, so I removed these lines in the new commit.

Reply:
BTW, thank you for your patience :)